### DISCUSSION TITLE
[功能请求]: 我想用vue做一个自定义的前端页面，有没有接口或者好的思路方向可以解决

### DISCUSSION THREAD

**Discussion by 120805481** [GaiZhenbiao/ChuanhuChatGPT#726] [General]
### 相关问题

我想用vue做一个自定义的前端页面，有没有接口或者好的思路方向可以解决

### 可能的解决办法

接入vue前端项目，目前gradio的方式有很多想做自定义调整的很麻烦

### 帮助开发

- [X] 我愿意协助开发！

**Comment by yanyuxiyangzk** [comment]
你可以用flask技术提供api接口，自定义前端页面，相当于重新开发了
